recipes/thrift-cpp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: thrift-cpp
+  version: "0.9.3"
+
+source:
+  fn: thrift-0.9.3.tar.gz
+  url: http://archive.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
+  md5: 88d667a8ae870d5adeca8cb7d6795442
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - boost
+    - autoconf
+    - automake
+    - libtool
+    - openssl
+    - bison
+    - flex
+    - pkg-config
+    - zlib
```

Comment:
Please pin `zlib` to `1.2*` here as described in the pinning [wiki]( https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies ).